File: patterns/c-patterns/pattern39.c
// Pattern 39: ...

// 1
// 2*
// 3*3
// 4*4*
// 5*5*5

#include <stdio.h>
int main()
{
    int i, j;
    int n = 5;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= i; j++)
        {
            if (j % 2 == 0)
                printf("*");
            else
                printf("%d", i);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern39.js
// Pattern 39: ...

// 1
// 2*
// 3*3
// 4*4*
// 5*5*5

const n = 5;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = 1; j <= i; j++) {
        if (j % 2 === 0) {
            row += '*';
        } else {
            row += i;
        }
    }

    console.log(row);
}
